### PR TITLE
Feat/card

### PR DESCRIPTION
**Pull request by Aditi0505**
# Description

-  Added Card Component
-  Changed class names in other components to avoid conflicts.

# Screenshots

image
image
image
image
image
image
image
image

### CODE REVIEW

File: components/card/card.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
+@import url("../basic.css");
+.card-container,
+.card-container-overlay,
+.text-card-container,
+.card-container-shadow {
+  width: auto;
+  max-width: 286px;
+  height: fit-content;
```
Comment (anupkgurung) [inline components/card/card.css:8]: px can be changed to % or rem.

File: components/card/card.css [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  border: 1px solid var(--secondary-color);
+}
```
Comment (rohanmathur91) [inline components/card/card.css:13]:
``` suggestion
You can have a common class called card, give this common styles to that class and use were needed.
```

**Review by anupkgurung — COMMENTED** [review]
Values are used in px. Would suggest to change it into % or rems.

File: components/card/card.css [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+  background-color: var(--accent-gray);
+  align-content: center;
+  justify-content: center;
+  font: var(--semibold) var(--sm-size) var(--body-font);
+  color: var(--primary-color);
+}
+.card-body-overlay,
+.card-image-overlay,
+.icons-overlay {
+  opacity: 70%;
```
Comment (rohanmathur91) [inline components/card/card.css:140]: As you are repeating this so you can have a utility class for opacity.

File: components/card/card.css [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+}
+.hide-card-body,
+.hide-icons,
+.hide-card-image,
+.hide-badge-horizontal {
+  display: none;
+}
```
Comment (rohanmathur91) [inline components/card/card.css:147]: same have a class e.g hide and use that

File: components/card/card.css [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+  border-style: none;
+  background-color: var(--primary-color);
+  font-size: var(--sm-size);
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (Naveenchandar) [inline components/card/card.css:121]: flex properties are repeated. You can move flex properties in utilities file and use the respective classes wherever needed.

File: docs/docs-style.css [hunk 6]
```diff
@@ -84,7 +86,8 @@ body {
 .badge-avatar-example,
 .badge-button-example,
 .badge-label-example,
-.button-example {
+.button-example,
+.example {
   display: flex;
   justify-content: center;
   align-items: center;
```
Comment (rohanmathur91) [inline docs/docs-style.css:93]: You can have a utility class for this eg flex.

**Review by rohanmathur91 — APPROVED** [review]
Added a few comments, the rest looks good.

File: components/card/card.css [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+  display: none;
+}
+.text-card-container {
+  border: 1px solid var(--secondary-color);
+  color: var(--accent-pink);
+  font: var(--regular) var(--md-size) var(--body-font);
+}
+
+.card-container-horizontal {
+  width: 600px;
```
Comment (Naveenchandar) [inline components/card/card.css:155]: Any specific reason to use px instead of relative units ?

File: components/card/card.css (hunk 2 above)
Comment (Aditi0505) [reply components/card/card.css:13]: Thanks, I will make the change.

File: components/card/card.css (hunk 1 above)
Comment (Aditi0505) [reply components/card/card.css:8]: Sure, I will change it to % or rem.

File: components/card/card.css (hunk 3 above)
Comment (Aditi0505) [reply components/card/card.css:140]: Sure, I will change this

File: components/card/card.css (hunk 5 above)
Comment (Aditi0505) [reply components/card/card.css:121]: Thanks, I will make the change.

File: components/card/card.css (hunk 7 above)
Comment (Aditi0505) [reply components/card/card.css:155]: No specific reason. I will change it to % or rem.

**Review by anupkgurung — APPROVED** [review]
looks good!!

**Pull request merged**